content/en/checkout/reference.md

```diff
@@ -151,13 +151,72 @@ An object with the following properties:
 
 #### Success handler:
 
-`onSuccess(checkoutPaymentInfo)`
-
-Receives an object with the following property:
-
-| Property | Type      | Description                                                                                                                                 |
-| -------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
-| `paid`   | `boolean` | Whether the payment was completed (for synchronous methods, e.g. credit card) or not (for asynchronous methods, e.g. Multibanco reference). |
+`onSuccess(successInfo)`
+
+Receives an object with the following properties:
+
+| Property  | Type      | Description                                                                        |
+| --------- | --------- | ---------------------------------------------------------------------------------- |
+| `id`      | `string`  | The id of the Checkout session.                                                    |
+| `type`    | `string`  | The payment type for this Checkout (`'single'`, `'frequent'` or `'subscription'`). |
+| `payment` | `Object`  | Detailed information about the payment.                                            |
+
+Properties of the `payment` Object:
+
+| Property     | Type      | Description                                                                      |
+| ------------ | --------- | -------------------------------------------------------------------------------- |
+| `id`         | `string`  | The payment's id.                                                                |
+| `method` (1) | `string`  | The payment method chosen by the customer.                                       |
+| `status` (2) | `Object`  | The status of the payment.                                                       |
+| `value`      | `number`  | The order value, rounded to two decimal places. Not used in `frequent` payments. |
+
```

Comment:
I would group here the `'Additional payment fields' ` table instead. 
Add the `method` column, and for properties like `id`, `method`, `status`, `value` you can set to `ALL` methods.
Then the numbered properties.
It's more clear.

Reply:
Good idea. Is it better like this?

Reply:
Much better. GJ!